tests/director/tests/runtime_api_test.go

```diff
@@ -745,6 +748,17 @@ func TestRuntimeTypeLabels(t *testing.T) {
 		runtimeTypeLabelValue, ok := runtime.Labels[conf.RuntimeTypeLabelKey].(string)
 		require.True(t, ok)
 		require.Equal(t, conf.KymaRuntimeTypeLabelValue, runtimeTypeLabelValue)
+
+		t.Log("Validate that the Application Namespace is available...")
+		appNamespace := runtime.ApplicationNamespace
+		require.NotNil(t, appNamespace)
+		require.Equal(t, conf.KymaApplicationNamespaceValue, *appNamespace)
+
+		t.Log("Validate that the region label of the runtime is available...")
+		rtRegionLabelValue, ok := runtime.Labels[tenantfetcher.RegionKey].(string)
+		require.True(t, ok)
+		require.Equal(t, conf.DefaultTenantRegion, rtRegionLabelValue)
+
```

Comment:
Could we also add "negative" assertions in the sub-test below that is registering the runtime without integration system credentials?